common/changes/@itwin/reports-config-widget-react/uyen-report-config-migration_2023-07-26-23-02.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/reports-config-widget-react",
+      "comment": "Migrated UI items for iTwin UI 2.x and iTwinJS 4.x upgrade.",
+      "type": "patch"
```

Comment:
this should be a 'minor' type change
'patch' implies there are no breaking changes, but we have a major dependency overhaul

Reply:
I just changed it to 'minor'.